src/main/java/frc/robot/Robot.java

```diff
@@ -92,25 +86,16 @@ public void robotPeriodic() {
    */
   @Override
   public void autonomousSetup() {
-    m_autoSelected = m_chooser.getSelected();
-    // m_autoSelected = SmartDashboard.getString("Auto Selector", kDefaultAuto);
-    System.out.println("Auto selected: " + m_autoSelected);
   }
 
   /**
    * This function is called periodically during autonomous.
+   * 
+   * @throws InterruptedException
    */
   @Override
-  public void autonomousRoutine() {
-    switch (m_autoSelected) {
-      case kCustomAuto:
-        // Put custom auto code here
-        break;
-      case kDefaultAuto:
-      default:
-        // Put default auto code here
-        break;
```

Comment:
You might want to consider putting back this `SendableChooser` stuff, even if there is only currently one autonomous routine implemented. It's a fairly reasonable way to go about having an easily selectable routine without having to redeploy code.

Reply:
I'll put it back in when we have multiple auto routines